plugins/content/adaptiveimage/adaptiveimage.php

```diff
@@ -74,6 +74,8 @@ protected function insertFocus(&$text, &$params)
 		// Match pattern and return array into $images
 		preg_match_all($searchImage, $text, $images);
 
+		$storage = new JSONFocusStore;
```

Comment:
Why the controller initiating a storage class? anyway lets keep this to make your project more simple (**for now**).
But in future milestones you need to do something like this

- Trigger an event on controller/model like "onFocusStoreSetup"
- Pass your {whatEver}Store to the event params
- Retrieve all the store classes
- Get the default one from the settings
- Select it as $storage = $storages->getStorage("json");
- use it as you do